### PR TITLE
Add Lua and fix typos

### LINKED ISSUE #3453: [Edit] Documentation: Style Guide
### Type of Edit (select all that apply)

Add new content (definitions, codeblocks, etc.)

### Description (optional)

Lua is now a supported language for code blocks. This should be included in the [Style Guide](https://github.com/Codecademy/docs/blob/main/documentation/style-guide.md#docs-style-guide).

Update the documentation/style-guide.md file to:
- include `lua` in the [Supported Code Blocks](https://github.com/Codecademy/docs/blob/main/documentation/style-guide.md#supported-code-blocks) section

### Code of Conduct

- [X] By submitting this issue, I agree to follow Codecademy Doc's [Code of Conduct](https://github.com/Codecademy/docs/blob/main/.github/CODE_OF_CONDUCT.md).

### For Maintainers

- [x] Labels added
- [x] Issue is assigned

### PR DESCRIPTION
**Pull request by cslylla**
### Description

- Update documentation/style-guide.md file to include Lua as a supported language and fix typos.
- Closes #3453 

### Type of Change

- Editing an existing entry (fixing a typo, bug, issues, etc)
- Updating the documentation

### Checklist

- [X] All writings are my own.
- [X] My entry follows the Codecademy Docs style guide.
- [ ] My changes generate no new warnings.
- [X] I have performed a self-review of my own writing and code.
- [X] I have checked my entry and corrected any misspellings.
- [X] I have made corresponding changes to the documentation if needed.
- [X] I have confirmed my changes are not being pushed from my forked `main` branch.
- [X] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [X] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

File: documentation/style-guide.md [hunk 1]
```diff
@@ -71,21 +71,21 @@ There are three types of Docs entries:
 ## Headers/Sections
 
 - Do not use any `h1` headers (in Markdown preceded by `#`).
-- Do not include a title header, that is automatically provided by the metadata.
+- Do not include a title header automatically provided by the metadata.
```
Comment (SSwiniarski) [inline documentation/style-guide.md:74]:
```suggestion
- Do not include a title header. It is automatically provided by the metadata.
```

File: documentation/style-guide.md [hunk 2]
```diff
@@ -178,18 +179,18 @@ Codebytes can be used with the following languages:
...
+- In some cases, linking out to external standards documents or entities may be appropriate. (Example: [XML 1.0 Specification](http://www.w3.org/TR/REC-xml), [World Wide Web Consortium](https://www.w3.org/standards/), [Red Hat Enterprise Linux](https://www.redhat.com/en/technologies/linux-platforms/enterprise-linux))
+- When linking items outside the document, use the full path. Use a relative path when linking to items inside the document (i.e., Headers).
 - If there are parentheses in the link, replace them with `%28` and `%29`.
 
 ## File Hosting
 
-- If you'd like to host files such as images, or GIFs, please upload it to the [/media folder](https://github.com/Codecademy/docs/tree/main/media).
-- Images should be linked to using the following url: "https://raw.githubusercontent.com/Codecademy/docs/main/media/" followed by the image name.
+- If you'd like to host files such as images or GIFs, please upload them to the [/media folder](https://github.com/Codecademy/docs/tree/main/media).
+- Images' `src` attribute should be linked to the URL: "https://raw.githubusercontent.com/Codecademy/docs/main/media/" followed by the image name.
```
Comment (SSwiniarski) [inline documentation/style-guide.md:191]:
```suggestion
- Images should be linked to the URL: "https://raw.githubusercontent.com/Codecademy/docs/main/media/" followed by the image name.
```
These are Markdown files, not HTML.

**Review by SSwiniarski — CHANGES_REQUESTED** [review]
@cslylla I did a first review and left some comments.

**Review by SSwiniarski — APPROVED** [review]
@cslylla Looks good for a second review.

**Review by yangc95 — APPROVED** [review]

**Pull request merged**